model_cards/SZTAKI-HLT/hubert-base-cc/README.md

```diff
@@ -0,0 +1,43 @@
+---
+language: hu
+license: apache-2.0
+datasets:
+- Common Crawl
+- Wikipedia
```

Comment:
```suggestion
- common_crawl
- wikipedia
```

if we want datasets to be linked to the ones listed in https://huggingface.co/datasets this is the currently required format